hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestScmChillMode.java

```diff
@@ -226,36 +213,36 @@ public void testIsScmInChillModeAndForceExit() throws Exception {
 
   }
 
-  @Test(timeout=300_000)
+  @Test(timeout = 300000)
```

Comment:
Idea was to run mini cluster in parallel, with linear execution tests will pass anyway. IMO this will not test event driven chill mode correctly.

Reply:
Sorry not got your comment clearly, as each test is run after one another, and is also in the test even if we start a thread we wait for using get().

And your comment is for the line timeout, I need to change the value back?

Sorry if I am missing something here.